pubspec.yaml

```diff
@@ -38,6 +38,7 @@ dependencies:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
+  fluttertoast: ^8.2.4
```

Comment:
As mentioned by @gnprice in [the stream](https://chat.zulip.org/#narrow/stream/48-mobile/topic/.23F465.20Show.20.22Connecting.E2.80.A6.22.20banner.20.28or.20equivalent.29.20when.20serv.2E.2E.2E/near/1763901) no need to use `fluttertoast`